unstructured/partition/json.py

```diff
@@ -6,7 +6,7 @@
 from unstructured.partition.common import exactly_one
 from unstructured.staging.base import dict_to_elements
 
-LIST_OF_DICTS_PATTERN = r"\A\s*\[\s*{"
+LIST_OF_DICTS_PATTERN = r"\A\s*\[\s*{?"
```

Comment:
why this change?

Reply:
It raised an error on empty lists before